lib.properties

```diff
@@ -1,5 +1,5 @@
-VERSION_NAME=1.0.0
-VERSION_CODE=1
+VERSION_NAME=1.1.0
```

Comment:
I would keep the version the same. Noone uses this now so it does not matter

Reply:
Is it OK when it's pushed to bintray+maven? Will they replace the current version?

Reply:
yes, you just need to refresh dependencies if you use it already in some project

Reply:
I'm not so sure. It looks like my GitHub Action ran before I incremented the version and it failed because the artifact name is the same.  
https://github.com/AckeeCZ/TextInputLayout/runs/487178480?check_suite_focus=true#step:8:52

The fact that the action was triggered seems to be another issue though :(